frontend/src/app/spindle/spindle.component.ts

```diff
@@ -59,8 +88,9 @@ export class SpindleComponent implements OnInit, OnDestroy {
             });
     }
 
-    parse() {
-        this.export('term-table');
+    parse(): void {
+        this.clearResults();
```

Comment:
The only substantial change I applied beyond your PR comments is to move `clearResults()` here from `export()`. When clicking one of the export buttons, the Term and proof/tex would disappear from the UI. Now, the Term is only reset when the user submits a new input.